// const promise =new Promise((reject, resolve)=> {
//     resolve("Done");
// });

// promise.then((data)=>console.log(data)).catch((err)=>console.log(err))

//-------------------------------------------------------------------------------------

// const promise = (a,b)=> new Promise((resolve, reject)=>{
//     return resolve("Hello");
// })

// const check = promise();
// check.then((data)=>console.log(data)).catch((err)=>console.log(err))
